manifests/b/belgium/eid/viewer/5.1.12-build6095/belgium.eid.viewer.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: belgium.eid.viewer
+PackageVersion: 5.1.12-build6095

```

```suggestion
PackageVersion: 5.1.6095
```
This should match what is written to control panel